src/python/readme.md

```diff
@@ -0,0 +1 @@
+This directory will hold the python files for managing directories or snaps, application of geim, and creating plots in 2D.
```

Comment:
This subfolder contains the python scripts.

Later, when files are added, you will add in this readme a short description of each file and its purpose